assembl/graphql/schema.py

```diff
@@ -345,6 +349,9 @@ class Mutations(graphene.ObjectType):
     delete_token_vote = DeleteTokenVote.Field()
     add_gauge_vote = AddGaugeVote.Field()
     delete_gauge_vote = DeleteGaugeVote.Field()
+    add_post_extract = AddPostExtract.Field()
+    update_extract = UpdateExtract.Field()
+    delete_extract = DeleteExtract.Field()
```

Comment:
You added those 3 mutations twice.

Reply:
Look at commit order, but I removed them.